kratos/geometries/quadrature_point_surface_in_volume_geometry.h

```diff
@@ -69,8 +70,11 @@ class QuadraturePointSurfaceInVolumeGeometry
         GeometryShapeFunctionContainerType& ThisGeometryShapeFunctionContainer,
         TangentMatrixType LocalTangents )
         : BaseType(ThisPoints, ThisGeometryShapeFunctionContainer)
-        , mLocalTangents(LocalTangents)
```

Comment:
I think you can still leave this here. Just only add the error message

Reply:
But then we might get a segmentation fault, before we catch the error.

Reply:
Hmm could be.